docs/app/docs/overlay/sonner/page.mdx

```diff
@@ -0,0 +1,132 @@
+                        Widget buildToast(BuildContext context, FToast toast) => IntrinsicHeight(
+                          child: FCard(
+                            style: cardStyle,
+                            title: const Text('Event has been created'),
+                            subtitle: const Padding(
+                              padding: EdgeInsets.symmetric(vertical: 5),
+                              child: Text('This is a brief description that provides additional context.'),
+                            ),
+                            child: FButton(onPress: () => toast.dismiss(), child: const Text('Undo')),
+                          ),
+                        );
+
+                        showFToast(context: context, builder: buildToast);
+                      },
+                      child: const Text('Show Small Toast'),
+                    ),
+                    FButton(
+                      intrinsicWidth: true,
+                      onPress: () {
+                        Widget buildToast(BuildContext context, FToast toast) => IntrinsicHeight(
+                          child: FCard(
+                            style: cardStyle,
+                            title: const Text('Event has been created'),
+                            subtitle: const Padding(
+                              padding: EdgeInsets.symmetric(vertical: 5),
+                              child: Text(
+                                'This is a more detailed description that provides comprehensive context and additional '
+                                'information about the notification, explaining what happened and what the user might expect '
+                                'next.',
+                              ),
+                            ),
+                            child: FButton(onPress: () => toast.dismiss(), child: const Text('Undo')),
+                          ),
+                        );
+
+                        showFToast(context: context, builder: buildToast);
+                      },
+                      child: const Text('Show Large Toast'),
+                    ),
+                  ],
+                ),
+              ),
+            ),
+          ),
+        );
+      }
+    }
+    ```
+
+  </Tabs.Tab>
+</Tabs>
+
+## CLI
+
+To generate and customize this style:
+
+```shell copy
+dart run forui style create sonner 
+```
+
+## Usage
+
+### `showFToast(...)`
+
+```dart copy
+showFToast(
+  context: context,
+  style: FSonnerStyle(...),
+  alignment: FSonnerAlignment.,
+  duration: const Duration(seconds: 10),
+  builder: (context, toast) => FCard(
+    title: Text('Download Complete'),
+    subtitle: Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      spacing: 10,
+      children: [
+        Text('Your file has been downloaded.'),
+        FButton(
+          onPress: () => toast.dismiss(),
+          child: const Text('Dismiss'),
+        ),
+      ],
+    ),
+  ),
```

Comment:
After some thinking, I think we should be the one providing the content for the user. Similar to the types in the original Sonner library and this should probably be like a raw content.

See https://sonner.emilkowal.ski/ types.